repository-data/application/src/main/resources/hcm-config/configuration/update/EximImport.yaml

```diff
@@ -2,12 +2,13 @@
 definitions:
   config:
     /hippo:configuration/hippo:update/hippo:registry/EximImport:
-      hipposys:batchsize: 10
+      hipposys:batchsize: 1
       hipposys:description: Exim import script modified by Valtech
       hipposys:dryrun: false
       hipposys:parameters: "{\r\n  \"fileInJson\": true,\r\n  \"sourceBaseFolderPath\"\
-        : \"file:${java.io.tmpdir}/exim-import/\"\r\n}"
+        : \"file:/tmp/migrator/exim-import/\"\r\n}"
```

Comment:
Comment just to double check this change with fixed path is what we expect. Do we have the same fixed path for each test and production environment?

Reply:
This file not part of the PR, so will remove.